### ISSUE TITLE
mailu 2.0.30 ate mail, unknown reason, might be misconfig

### ISSUE THREAD

**Issue by christf** [Mailu/Mailu#2994]
I am using mailu 2.0.30 from helm chart and recently upgraded from 2.0.29. Since the upgrade mails were fetched with fetchmail, removed from the upstream account but never hit the mailbox in dovecot. This may entirely be my fault due to misconfiguration. In the meantime I disabled fetchmail for now to avoid any further mail disappear.  

At this point I am looking for help to identify the problem. What should I check?

**Comment by ghostwheel42** [comment]
You should look in the fetchmail and dovecot logs.
Fetchmail should take care not to delete the source until it has delivered to the target.

**Comment by christf** [comment]
the mailu-admin pod is not coming up, restarting due to liveness check / readiness check fail. There is no error message in the admin container. I can see attempts to connect to the admin pod by other containers which then of course fail.

**Comment by ghostwheel42** [comment]
I'm sorry - i can't help you with deployments not generated using [setup.mailu](https://setup.mailu.io/)

**Comment by guest42069** [comment]
Check that your kubernetes is creating a /.dockerenv file in your `admin` container, otherwise their provided helm charts won't work and also won't correctly log the errors. Technically it seems that the helm charts they provide are no longer supported, only the docker-compose method is.

https://github.com/Mailu/Mailu/commit/2a570d0f6f9f5d799ffcaaf68963e729830e03b9

**Comment by ghostwheel42** [comment]
> Technically it seems that the helm charts they provide are no longer supported, only the docker-compose method is.

Yes, there are is another repo / people for some helm charts using the containers from the Mailu docker compose stack.
Also the matrix channels are a good way to find people running Mailu in kubernetes.

**Comment by guest42069** [comment]
Alternatively, just add `I_KNOW_MY_SETUP_DOESNT_FIT_REQUIREMENTS_AND_WONT_FILE_ISSUES_WITHOUT_PATCHES` to your environment variables and it'll work fine again if it was before, at least until you can migrate to a different software stack.

**Comment by nextgens** [comment]
> Alternatively, just add `I_KNOW_MY_SETUP_DOESNT_FIT_REQUIREMENTS_AND_WONT_FILE_ISSUES_WITHOUT_PATCHES` to your environment variables and it'll work fine again if it was before, at least until you can migrate to a different software stack.

I don't understand what you are going on about. We make it explicitly clear what is and what is not supported... and despite our best efforts, we keep getting issues filed in the wrong places for things we do not support. The released version of the helm-charts deploys a build of Mailu that works, the upcoming version will also do so. Kubernetes without helm-charts is not supported as it is fairly clear from https://mailu.io/2.0/kubernetes/mailu/index.html
